MatrixSDK/MXSession.h

```diff
@@ -1087,7 +1087,7 @@ typedef void (^MXOnBackgroundSyncFail)(NSError *error);
 
  @return the MXRoomSummary instance.
  */
-- (MXRoomSummary *)roomSummaryWithRoomId:(NSString*)roomId;
+- (nullable MXRoomSummary *)roomSummaryWithRoomId:(NSString*)roomId;
```

Comment:
I'm pretty sure this nullability specifier will trigger a ton of the warnings for the rest of the methods in this file. Have a look at `MX_ASSUME_MISSING_NULLABILITY_BEGIN`

Reply:
Oh we don't have it in MXSession.h